twelveDaysOfChritmas.pi

```diff
@@ -0,0 +1,70 @@
+Speical version of Linux, since Chrismas is cuming.
```

Comment:
> cuming

Yeah, i'd doubt they'll put Christmas stuff in Linux.

Reply:
Lmao

Reply:
I did not intend for the word to sound like that tho lmao